.gitignore

```diff
@@ -216,8 +216,8 @@ FakesAssemblies/
 **/*.Server/ModelManifest.xml
 _Pvt_Extensions
 
-# App_Data folder
-**/App_Data
+# Data folder

```

Comment:
@marktaling Since we're no longer tied to how IIS works, I've renamed the folder for storing the data to just data. What do you think?

Reply:
I think that's probably for the best.